### ISSUE TITLE
Need to interact with particular particle on mouse touch and destroy it

### ISSUE THREAD

**Issue by DeepBhas** [pixijs/particle-emitter#199]
Hi, 

I'm new to pixi-particles library, I'm using pixi-particles v4:3:0 and pixi-js-legacy 5:3:7

I creating a scenario using pixi-particles where I integrated a rain effect. I need to destroy particular particle emit from emitter on mouse interact. How can we achieve this. I'm facing difficulty to get particular particle object on mouse interact.

Much needed help ASAP.

Thanks

**Comment by andrewstart** [comment]
I would advise against this, as hit testing against hundreds/thousands of particles each mouse down/up will not be good for performance. If you still want to do it, then I would write your own [behavior implementation](https://github.com/pixijs/particle-emitter/blob/master/src/behaviors/Behaviors.ts) that adds the mouse event listeners and reports the results. I haven't used the new event library in PixiJS v7 yet, but I believe it bubbles, so you could also try adding a listener to the parent of the particles and the event should tell you which object is the target - you may need to make a behavior to make particles interactive so that they are valid as individual targets.

**Comment by DeepBhas** [comment]
Yes @andrewstart I need to achieve the same functionality using mouse events ( over / left click ). Please do the needful.
